vision_datasets/kv_pair/manifest.py

```diff
@@ -0,0 +1,127 @@
+from typing import Optional
+from ..common import ImageLabelManifest, AnnotationWiseDatasetManifest, DatasetTypes
+        
+
+class FieldSchema:
+    TYPE_NAME_TO_PYTHON_TYPE = {
+        'array': list,
+        'object': dict,
+        'number': (float, int),
+        'integer': int,
+        'boolean': bool,
+        'string': str, 
+        'boundingBox': list
```

Comment:
We could even make it a `list[float] | list[int]`.

Reply:
I haven't done this yet, since the parameterized type is not supported by `isinstance` kind of check, and we haven't finalized the design here. Will improve it in later PR